packages/manager/modules/sms/src/sms/compose/index.js

```diff
@@ -1,23 +1,30 @@
 import angular from 'angular';
-import controller from './telecom-sms-sms-compose.controller';
-import template from './telecom-sms-sms-compose.html';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+
+import {
+  component,
+  name as componentName,
+} from './telecom-sms-sms-compose.component';
+
+import {
+  state,
+  name as stateName,
+} from './telecom-sms-sms-compose.state';
 
 import './addPhonebookContact/telecom-sms-sms-compose-addPhonebookContact.less';
 
 const moduleName = 'ovhManagerSmsSmsCompose';
 
-angular.module(moduleName, []).config(($stateProvider) => {
-  $stateProvider.state('sms.service.sms.compose', {
-    url: '/compose',
-    views: {
-      'smsView@sms.service': {
-        template,
-        controller,
-        controllerAs: 'SmsComposeCtrl',
-      },
-    },
-    translations: ['../../../sms/dashboard', '.'],
-  });
-});
+angular
+  .module(moduleName, [
+    'ngOvhTelecomUniverseComponents',
+    'ui.router',
+  ])
+  .component(componentName, component)
```

Comment:
According to my previous comment, it could be:

```suggestion
  .component('smsCompose', component)
```